### DISCUSSION TITLE
About pretrained spynet in BasicVSR

### DISCUSSION THREAD

**Discussion by JingYansen** [open-mmlab/mmediting#1906] [Models]
Is the spynet pretrained from images? Notice that in BasicVSR the spynet is used on the features instead of images, I am wondering the how to guarantee that the spynet pretrained from image can be generalized to features.

**Comment by Z-Fran** [comment]
https://github.com/open-mmlab/mmagic/blob/main/mmagic/models/editors/basicvsr/basicvsr_net.py#L104   The spynet is used on the images here.
